packages/api-logs/main.css

```diff
@@ -52,8 +48,27 @@
 .hub-reference .logs .logs-list table td,
 .hub-reference .logs .logs-list table th {
   padding: 10px;
-  word-break: break-all;
+  white-space: nowrap;
   overflow-x: hidden;
+  text-overflow: ellipsis;
+}
+.hub-reference .logs .logs-list table th.method {
+  width: 7%;
```

Comment:
Possibly? I think in this case it's fine, since it's really the proportions that are important. As long as the url one is the widest for example it should look good